promql/promqltest/testdata/at_modifier.test

```diff
@@ -91,7 +91,8 @@ eval instant at 25s metric{job="1"} @ 50 + metric{job="1"} @ 100
   {job="1"} 15
 
 # Note that this triggers an info annotation because we are rate'ing a metric that does not end in `_total`.
-eval_info instant at 25s rate(metric{job="1"}[100s] @ 100) + label_replace(rate(metric{job="2"}[123s] @ 200), "job", "1", "", "")
+eval instant at 25s rate(metric{job="1"}[100s] @ 100) + label_replace(rate(metric{job="2"}[123s] @ 200), "job", "1", "", "")
+  expect info
```

Comment:
I would leave out this line and the comment above (line 93). We essentially only had to add this because `eval` was originally too strict about annotations. Now we can remove both again.